app/questionnaire/value_source_resolver.py

```diff
@@ -180,29 +224,18 @@ def resolve(
     ) -> Union[ValueSourceEscapedTypes, ValueSourceTypes]:
```

Comment:
While method is being changed - Union[ValueSourceEscapedTypes, ValueSourceTypes] ->ValueSourceEscapedTypes | ValueSourceTypes

Reply:
Updated this 👍